docs/BR.md

```diff
@@ -3147,45 +3174,88 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 
 ## 7.2 CRL profile
 
+Prior to 2024‐03‐15, the CA SHALL issue CRLs in accordance with the profile specified in these Requirements or the profile specified in Version 1.8.7 of the Baseline Requirements for the Issuance and Management of Publicly‐Trusted Certificates. Effective 2024‐03‐15, the CA SHALL issue CRLs in accordance with the profile specified in these Requirements.
+
 ### 7.2.1 Version number(s)
 
+Certificate Revocation Lists MUST be of type X.509 v2.
+
 ### 7.2.2 CRL and CRL entry extensions
 
-1. `reasonCode` (OID 2.5.29.21)
+If the CA asserts compliance with these Baseline Requirements, all CRLs that it issues MUST comply with the following CRL profile, which incorporates, and is derived from [RFC 5280](https://tools.ietf.org/html/rfc5280). Except as explicitly noted, all normative requirements imposed by RFC 5280 shall apply, in addition to the normative requirements imposed by this document. CAs SHOULD examine [RFC 5280, Appendix B](https://tools.ietf.org/html/rfc5280#appendix-B) for further issues to be aware of.
+
+A full and complete CRL is a list of all revoked certificates issued by the CA for any and all reasons.
+
+A partitioned CRL (sometimes referred to as a "sharded CRL") is a list of revoked certificates issued by the CA for any and all reasons constrained to a specific scope (e.g., temporal sharding).
+
+Minimally, CAs MUST issue either a "full and complete" CRL - or "partitioned" CRLs. Aside from the presence of the `IssuingDistributionPoint` extension (OID 2.5.29.28) in partitioned CRLs, both CRL formats are syntactically the same from the perspective of this profile.
+
+If using only partitioned CRLs, the full set of partitioned CRLs MUST cover the complete set of public-key certificates issued by the CA. Thus, the complete set of partitioned CRLs MUST be equivalent to a full CRL for the same set of public-key certificates, if the CA was not using partitioned CRLs. 
```

Comment:
This is wildly editorial, so feel free to ignore, but I would combine/rephrase/reorder these paragraphs. I works better for my brain this way, but that's definitely not a claim that my way works better for everyone or even a majority of people:

```suggestion
A full and complete CRL is CRL whose scope includes all certificates issued by the CA.

A partitioned CRL (sometimes referred to as a "sharded CRL") is a CRL with a a constrained scope, such as all certificates issued by the CA during a certain period of time ("temporal sharding"). Aside from the presence of the `IssuingDistributionPoint` extension (OID 2.5.29.28) in partitioned CRLs, both CRL formats are syntactically the same from the perspective of this profile.

Minimally, CAs MUST issue either a "full and complete" CRL or a set of "partitioned" CRLs which cover the complete set of certificates issued by the CA. In other words, if issuing only partitioned CRLs, the combined scope of those CRLs must be equivalent to that of a full and complete CRL. 
```

Reply:
Accepted suggestion with minor tweaks.  https://github.com/ryancdickson/staging/commit/12c2db11346481fd8efd150144692950827f968f